config/routes.rb

```diff
@@ -5,6 +5,11 @@
     get "/login", to: "login_session#new"
     post "/login", to: "login_session#create"
     delete "/logout", to: "login_session#destroy"
+    get "/index", to: "products#index"
```

Comment:
?

Reply:
routes này cho user xem toàn bộ sản phẩm ạ

Reply:
k để tên kiểu này nhé e
sao k dùng /products như mặc định

Reply:
Em update lai roi a